### ISSUE TITLE
Consider tooltip for roads

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#5162]
Right now to discover roads you need to hold space and be aware what T/K/H/S means:
![image](https://github.com/ZeroK-RTS/Zero-K/assets/2573076/698d1144-38f2-4608-8181-a2ba8ae89fae)

Consider making it so that roads don't require holding space, and having a less cyptic tooltip. Mockup:
![image](https://github.com/ZeroK-RTS/Zero-K/assets/2573076/7429fcec-1ba4-48bd-885c-932ff82d473d)

**Comment by GoogleFrog** [comment]
Seems good. Do roads exist by default though?

**Comment by sprunk** [comment]
Yes. Unequal roads are removed but if T==K==H==S then the road is kept.
